docs/aop/dev/prerequisites-for-adding-apps/prerequisites-for-adding-user-consent-management.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Prerequisites for adding user consent management
+description: Learn about the steps you need to take before you can start working with consent managers through AOP.
+template: howto-guide-template
+---
+
+This document describes the prerequisites for using consent managers through AOP
+
+### Required modules:
+
+* spryker/cms-slot
+* spryker/asset-external
+* spryker/asset-external-storage
+* spryker-shop/asset-external-widget
+* spryker/event
+* spryker/event-aws-sns-broker (?)
+
+### Event listeners:
+
+* AssetExternalStorageEventSubscriber (`\Spryker\Zed\AssetExternalStorage\Communication\Plugin\Event\Subscriber\AssetExternalStorageEventSubscriber`)
+* AssetExternalSubscriber (`\Spryker\Zed\AssetExternal\Communication\Plugin\Event\Subscriber\AssetExternalSubscriber`)
+
+### Rabbit MQ queues:
+
+* sync.storage.cms
+
+### CMS Slots:
+
+* external-asset-header-top
```

Comment:
is it hadrcoded  in the code? or provided in event?

Reply:
app-store-suite sends an event with this CMS slot, it should be added on a project level beforehand via data import

Reply:
ok is it hardcoded on PBC side? or user have to enter in manually in config form?

Reply:
it's hardcoded on PBC side

Reply:
ok